src/deluge/processing/stem_export/stem_export.h

```diff
@@ -81,7 +81,7 @@ class StemExport {
 	Error getUnusedStemRecordingFilePath(String* filePath, AudioRecordingFolder folder);
 	Error getUnusedStemRecordingFolderPath(String* filePath, AudioRecordingFolder folder);
 	int32_t highestUsedStemFolderNumber;
-	String lastSongNameForStemExport;
+	String lastFolderNameForStemExport;
```

Comment:
does this need to be initialized to an empty string?

Reply:
> does this need to be initialized to an empty string?

Yes that's a good idea! Done thx!